releases: v4.0 release notes (#171)
inveniosoftware/docs-invenio-rdm: docs/releases/versions/version-v4.0.0.md

# InvenioRDM v4.0

We're happy to announce InvenioRDM v4.0!

!!! Warning

    InvenioRDM v4.0 is **NOT** yet suitable for production services.

## Try it

Want to try the new features in v4.0? Just head over to our demo site: [https://inveniordm.web.cern.ch](https://inveniordm.web.cern.ch)

If you want to install it, follow the installation instructions on [https://inveniordm.docs.cern.ch/install/](https://inveniordm.docs.cern.ch/install/)

## What's new?

### Single sign-on (SSO)

InvenioRDM can now easily integrate with external authentication providers. You can enable ORCID integration and OpenID Connect/OAuth with Keycloak servers.

![](v4.0/sso1.png)

Read the [Authentication](/customize/authentication/) documentation section to get to know how to set it up.

SAML integration is also supported but it requires a more advanced setup.

#### Disable local login

InvenioRDM v4.0 comes with various new features around authentication.

You can now e.g. disable local authentication (login and registration) via a configuration variable.

![](v4.0/no-local.png)

> Changes to the user profile (change full name or e-mail address) can be blocked, which is particularly useful when use only external authentication in a is setup.

Reviewer: ```suggestion
Changes to the user profile (change full name or e-mail address) can be blocked, which is particularly useful when using only external authentication in your setup.
```
